### DISCUSSION TITLE
CSSNano + PostCSS Now Possible?

### DISCUSSION THREAD

**Discussion by anthonyp-usafacts** [jaredpalmer/razzle#1648] [Ideas & RFCs]
According to the code:

https://github.com/jaredpalmer/razzle/blob/eafcd1453a97b3b3978c49230177cabb6df6f3d8/packages/razzle/config/loadRazzleConfig.js#L15

Well, CSSNano [moved over that line in late May](https://github.com/cssnano/cssnano/pull/1135/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R53).

Razzle can now swap over to cssnano 🎉 

Are there other blockers for this that you know of?

**Comment by fivethreeo** [comment]
the new css optimiser is webpack5 only ..

https://github.com/jaredpalmer/razzle/issues/1641
